src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,206 @@
+import { useEffect, useState } from 'react';
+
+import { useParams, useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
 
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+type PageState =
+  | 'loading'
+  | 'error'
+  | 'nothingFound'
+  | 'changeFilter'
+  | 'showTable';
```

Comment:
it'll work, but it's not a common practice
enum is THE common approach for such

```suggestion
enum PageState {
  Loading,
  // or `Loading = 'loading'` if you rely on text value, otherwise enum items are indexed
  ...;
```